app/manifest/_base.json

```diff
@@ -1,8 +1,7 @@
 {
   "author": "https://metamask.io",
   "background": {
-    "page": "background.html",
-    "persistent": true
+    "service_worker": "app-init.js"
   },
   "browser_action": {
     "default_icon": {
```

Comment:
Manifest in this PR has minimal changes to just be able to run service worker, there is a separate task to do other changes in manifest.